docs/secure-connections/sauce-connect/proxy-tunnels.md

```diff
@@ -36,20 +37,17 @@ You can launch a new tunnel from the command line of the machine where the Sauce
 
 See [Basic Setup](/secure-connections/sauce-connect/setup-configuration/basic-setup) for full instructions on launching tunnels.
 
-### Stopping an Individual Tunnel via the Command Line
+### Stopping a Tunnel via Command Line
+There are two ways to do this:
+
+#### Method 1: `ctrl-c`
 Once Sauce Connect has been terminated (typically via `ctrl-c`), a call will be made from Sauce Connect to the REST API with instructions to terminate the Tunnel VM. Sauce Connect will continue to poll the REST API until the Tunnel VM has been halted and deleted.
 
 :::note
-If you are using the [High Availability Setup](/secure-connections/sauce-connect/setup-configuration/high-availability) and attempt to terminate a running test with `ctrl-C`, you will see a message that Sauce Connect Proxy will not terminate until tests have completed. If you want to force Sauce Connect Proxy to terminate before the test finishes, enter 'ctrl-C' again to force it to quit.
+If you are using the [High Availability Setup](/secure-connections/sauce-connect/setup-configuration/high-availability) and attempt to terminate a running test with `ctrl-c`, you will see a message that Sauce Connect Proxy will not terminate until tests have completed. If you want to force Sauce Connect Proxy to terminate before the test finishes, enter 'ctrl-C' again to force it to quit.
 :::
```

Comment:
This happens whenever there is an attempt to terminate a tunnel running a test, it's not high-availability specific.

Reply:
Something like: If you attempt to terminate a Sauce Connect Proxy that is running a test...